### ISSUE TITLE
Background shadow color

### ISSUE THREAD

**Issue by caelinsutch** [artrmz/flutter_control_pad#27]
Is there a way to change the background shadow color?

**Comment by artur-ios-dev** [comment]
No, not right now. It should be quite easy to add though, just need to add `List<BoxShadow> boxShadow` parameter to each factory method in the `CircleView` class as well as to `JoystickView`. Feel feee to open PR.

**Comment by caelinsutch** [comment]
In general I feel like there should be more control and/or better documentation for styling this widget. I'll open up a PR at some point when I'm not so busy at work.

**Comment by artur-ios-dev** [comment]
Sounds good, thank you! :)
